data_model/1.4/clusters/EnergyEVSE.xml

```diff
@@ -240,7 +240,6 @@ Davis, CA 95616, USA
     <struct name="ChargingTargetScheduleStruct">
```

Comment:
EVSE PREF should not be P

Reply:
this has been resolved now with the latest spec scrape.